### ISSUE TITLE
Update to Gradle 6.7

### ISSUE THREAD

**Issue by Mazorius** [klaxit/hidden-secrets-gradle-plugin#9]
You are using Gradle 6.1.1 inside your plugin.

Think you should update to 6.7 and change to use [Java Toolchain](https://docs.gradle.org/current/userguide/toolchains.html).

With this you do not need the Java configuration any more.

**Comment by ben-j69** [comment]
Thanks @Mazorius for the suggestion

It looks like we could remove 
```
configure<JavaPluginConvention> {
    sourceCompatibility = JavaVersion.VERSION_1_8
}
```

Can you provide more details on what changes we could make on the project with this update ?

**Comment by Mazorius** [comment]
I will prepare a PR at the next days.

Are you using IntelliJ IDEA? If yes, can you please generate a `.editorconfig` file in the project root?
Than auto formation in IDEA will use your configuration for that.

**Comment by ben-j69** [comment]
@Mazorius `.editorconfig` added in https://github.com/klaxit/hidden-secrets-gradle-plugin/pull/8 👍

**Comment by ben-j69** [comment]
Hello @Mazorius this change has been made in the last PR : https://github.com/klaxit/hidden-secrets-gradle-plugin/pull/33 can you check if everything is correct as you suggested ?

Thanks

**Comment by Mazorius** [comment]
I think this is no solved with #33 👍

**Comment by ben-j69** [comment]
Done in https://github.com/klaxit/hidden-secrets-gradle-plugin/pull/33
